.github/workflows/bump_packages.yml

```diff
@@ -2,6 +2,12 @@ name: Update packages
 
 on:
   workflow_dispatch:
+    inputs:
+      filter:
+        type: string
+        description: A filter to apply on package names
```

Comment:
Package names have different structure between Debian and EL. Should this filter rather be on the "source" name?
So, passing `foreman_maintain` here will update `ruby-foreman-maintain` on Debian and `rubygem-foreman_maintain` on RPM?

Reply:
I think so. In the update scripts we know the gem name so we can match against that